cmd/shared.go

```diff
@@ -107,3 +108,10 @@ func requiredInputCheck(fieldsAndValues ...string) error {
 	}
 	return nil
 }
+
+// Outputs the message in a way that can be captured by testing
+func handleNilResults(message string, cmd *cobra.Command, fields ...interface{}) {
```

Comment:
if this returned an error, it could just `return nil` at the end, then everywhere it is used can just do this
```
return handleNilResults("No associated users found for organization '%s'\n", cmd, organizationName)
```